crates/bevy_render/src/render_phase/mod.rs

```diff
@@ -84,9 +90,17 @@ impl<I: PhaseItem> RenderPhase<I> {
         let mut draw_functions = draw_functions.write();
         draw_functions.prepare(world);
 
-        for item in &self.items {
-            let draw_function = draw_functions.get_mut(item.draw_function()).unwrap();
-            draw_function.draw(world, render_pass, view, item);
+        let mut index = 0;
```

Comment:
Curious what the assembly looks like for this block.

Reply:
Here's some annotated output from just the main render loop: [render.txt](https://github.com/bevyengine/bevy/files/12414752/render.txt)
And here's the full asm for the function: [render_full.txt](https://github.com/bevyengine/bevy/files/12414757/render_full.txt)
(had to use txt files as github complains about .asm)

Reply:
Thanks. Don't think it's a huge problem then. Branch predictor probably handles it fine.